src/assignment/views/AssignmentEdit.tsx

```diff
@@ -674,6 +693,45 @@ const AssignmentEdit: FunctionComponent<DefaultSecureRouteProps<{ id: string }>>
 					}}
 				/>
 			)}
+
+			{assignment && (
+				<AddBlock
+					isOpen={!!isAddBlockOpen}
```

Comment:
same remark as the other PR
You can use the addBlock object as the boolean to open the modal, so you don't need the useSingleEntityModal hook: https://github.com/viaacode/avo2-client/pull/1366#discussion_r908250134